library/dd-sdk-android-session-replay/build.gradle.kts

```diff
@@ -78,6 +78,7 @@ dependencies {
     implementation(libs.kotlin)
     implementation(libs.gson)
     implementation(libs.androidXAppCompat)
+    implementation(libs.googleMaterial)
```

Comment:
doesn't it then make `dd-sdk-android-session-replay-material` module useless? I though the purpose of that module was to avoid having `material` dependency in the main module.

Reply:
ohhhh sorry that was a leftover 🤦